diff -uraN pal-954ff050fe7e3842947a0d00d619c0f3ba5e9cbd/src/core/imported/rdf/rdf/inc/amdrdf.h omc-pal-954ff050fe7e3842947a0d00d619c0f3ba5e9cbd/src/core/imported/rdf/rdf/inc/amdrdf.h
--- pal-954ff050fe7e3842947a0d00d619c0f3ba5e9cbd/src/core/imported/rdf/rdf/inc/amdrdf.h	2023-05-18 17:48:26.000000000 +0200
+++ omc-pal-954ff050fe7e3842947a0d00d619c0f3ba5e9cbd/src/core/imported/rdf/rdf/inc/amdrdf.h	2023-05-22 17:49:30.750126289 +0200
@@ -26,6 +26,7 @@
 
 #include <cstddef>
 #include <cstring>
+#include <cstdint>
 #include <functional>
 #include <memory>
 #include <stdexcept>
