diff -uraN pal-f80b95b308a42eb8f54d22b14961f3ea53674584/src/util/lnx/lnxUuid.cpp omv-pal-f80b95b308a42eb8f54d22b14961f3ea53674584/src/util/lnx/lnxUuid.cpp
--- a/pal-f80b95b308a42eb8f54d22b14961f3ea53674584/src/util/lnx/lnxUuid.cpp	2022-08-01 07:41:43.000000000 +0200
+++ b/omv-pal-f80b95b308a42eb8f54d22b14961f3ea53674584/src/util/lnx/lnxUuid.cpp	2022-08-16 16:02:25.092954437 +0200
@@ -30,6 +30,7 @@
 
 #include <random>
 #include <unistd.h>
+#include <time.h>
 
 namespace Util
 {
