diff -up amdvlk-vulkan-driver-2021.Q1.1/pal-1e61f9ab6eb2041c73d7e7bd8f2021fa05a4bdd8/inc/util/palSysUtil.h.omv~ amdvlk-vulkan-driver-2021.Q1.1/pal-1e61f9ab6eb2041c73d7e7bd8f2021fa05a4bdd8/inc/util/palSysUtil.h
--- amdvlk-vulkan-driver-2021.Q1.1/pal-1e61f9ab6eb2041c73d7e7bd8f2021fa05a4bdd8/inc/util/palSysUtil.h.omv~	2021-01-11 01:35:49.827232954 +0000
+++ amdvlk-vulkan-driver-2021.Q1.1/pal-1e61f9ab6eb2041c73d7e7bd8f2021fa05a4bdd8/inc/util/palSysUtil.h	2021-01-11 02:10:40.807109152 +0000
@@ -34,7 +34,7 @@
 #include "palUtil.h"
 #include <atomic>
 
-#if   defined(__unix__)
+#if   defined(__unix__) && (defined(__x86_64__) || defined(__i386__))
 #include <cpuid.h>
 #endif
 
@@ -462,10 +462,11 @@ PAL_INLINE void CpuId(
     uint32* pRegValues,
     uint32 level)
 {
-#if   defined(__unix__)
+#if   defined(__unix__) && (defined(__x86_64__) || defined(__i386__))
     __get_cpuid(level, pRegValues, pRegValues + 1, pRegValues + 2, pRegValues + 3);
 #else
-#error "Not implemented for the current platform"
+#warning "Not implemented for the current platform, assuming worst case"
+    *pRegValues = 0;
 #endif
 }
 
@@ -479,10 +480,11 @@ PAL_INLINE void CpuId(
     uint32 level,
     uint32 sublevel)
 {
-#if   defined(__unix__)
+#if   defined(__unix__) && (defined(__x86_64__) || defined(__i386__))
     __cpuid_count(level, sublevel, *pRegValues, *(pRegValues + 1), *(pRegValues + 2), *(pRegValues + 3));
 #else
-#error "Not implemented for the current platform"
+#warning "Not implemented for the current platform, assuming worst case"
+    *pRegValues = 0;
 #endif
 }
 
