diff -uraN pal-f37604d37883165c21b6e929a87936d4a39392ba/src/util/lnx/lnxUuid.cpp omv-pal-f37604d37883165c21b6e929a87936d4a39392ba/src/util/lnx/lnxUuid.cpp
--- pal-f37604d37883165c21b6e929a87936d4a39392ba/src/util/lnx/lnxUuid.cpp	2022-08-01 07:41:43.000000000 +0200
+++ omv-pal-f37604d37883165c21b6e929a87936d4a39392ba/src/util/lnx/lnxUuid.cpp	2022-08-16 16:02:25.092954437 +0200
@@ -30,6 +30,7 @@
 
 #include <random>
 #include <unistd.h>
+#include <time.h>
 
 namespace Util
 {
